sdk/storage/storage-blob/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 12.2.0 (2020.06)
 
 - Supported quick query. Added a new API `BlockBlobClient.query()`.
+- Increased the maximum block size for Block Blob from 100MiB to 4000MiB(~4GB). And thereby supporting ~200TB maximum size for Block Blob.
```

Comment:
Does it generally fit to every storage account? Or it only fit to some storage account with feature registered, or there other switch to control the feature. If so let's call it out in the comment of function(now), and later in sample.

The problem here is, user may try to set big block size, while it will fail if the feature is not enabled.(considering the feature is also in preview?)

Reply:
For all accounts.